docusaurus/docs/introduction/plugin-types-usage.md

```diff
@@ -0,0 +1,108 @@
+---
+id: plugin-types-usage
+title: Plugin types and usage
+description: Learn about the different types of Grafana plugins, their usage and configuration.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - usage
+  - provisioning
+  - provision
+  - configuration
+  - configure
+sidebar_position: 1
+---
+
+# Grafana plugin types and usage
+
+:::note
+
+This documentation discusses plugin configuration, provisioning, and usage for plugin developers. General information about plugin administration is available at [Plugin management](https://grafana.com/docs/grafana/latest/administration/plugin-management/).
+
+::: 
+## Overview
+
+Grafana plugin development allows for many options depending on the type of user experience you want to create. Whatever your need, there's a SDK-supported plugin type for your intended use:
+- **Data-source plugin** - a connection to a new database or other source of data,
+- **App plugin** - an integrated out-of-the-box experience, or
+- **Panel plugin** - a new way of visualizing data.
```

Comment:
Nit. Not sure if there's any particular reason of the ordering? Personally feels like I'm more used to listing panel plugins first, like here https://grafana.com/docs/grafana/next/administration/plugin-management/. No strong opinions though.

Reply:
The only reason for the ordering is that I followed the order listed in the ProductDNA document. But I agree that consistency with the Administration topic is a good idea.

Reply:
Just realize that the order is different below as well so would suggest changing the order here before merge. Order below
- Panel (visualization) plugins
- Data source plugins
- App plugins

Reply:
good catch, fixed.